File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
        int answer = n;
        for (char con : control.toCharArray()) {
            if (con == 'w') {
                answer += 1;
            }
            else if (con == 'a') {
                answer -= 10;
            }
            else if (con == 's') {
                answer -= 1;
            }
            else {
                answer += 10;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    var answer = n;
    const move = {
        'w' : 1,
        'a' : -10,
        's' : -1,
        'd' : 10
    };
    for (let con of control) {
        answer += move[con];
    }
    return answer;
}